GL_Framework/configuration/config.py

```diff
@@ -0,0 +1,86 @@
+import os
+
+import constants as const
+from dataclasses import dataclass, asdict
+import yaml
+from configuration.local_config import LocalConfig
+
+
+# local config stored not in variables but in json file
+@dataclass
+class ConfigModel:
```

Comment:
I suggest adding configuration YAML files per environment (prod, qa, dev/local), use one of them e. g. dev/local for default values and init the ConfigModel with these default values, and get rid of local_config.py